hubblestack_nova_profiles/cis/ubuntu-1604-level-1-scored-v1-0-0.yaml

```diff
@@ -702,29 +928,42 @@ stat:
           group: root
           uid: 0
           user: root
-          tag: CIS-3.1
-    description: Set User/Group Owner on bootloader config
+          tag: CIS-1.4.1
```

Comment:
These 2 (1.4.1) should be combined into a single check. `mode`, `allow_more_strict`, `uid`, `gid`, etc can all be in the same check.

Reply:
Done